docs/zh/14-reference/05-taosbenchmark.md

```diff
@@ -227,45 +227,34 @@ taosBenchmark -A INT,DOUBLE,NCHAR,BINARY\(16\)
 
 #### 数据库相关配置参数
 
-创建数据库时的相关参数在 json 配置文件中的 `dbinfo` 中配置，具体参数如下。这些参数与 TDengine 中 `create database` 时所指定的数据库参数相对应。
+创建数据库时的相关参数在 json 配置文件中的 `dbinfo` 中配置，个别具体参数如下。其余参数均与 TDengine 中 `create database` 时所指定的数据库参数相对应，详见[../../taos-sql/database]
 
 - **name** : 数据库名。
 
 - **drop** : 插入前是否删除数据库，默认为 true。
 
-- **replica** : 创建数据库时指定的副本数。
+#### 流式计算相关配置参数
 
-- **days** : 单个数据文件中存储数据的时间跨度，默认值为 10。
+创建流式计算的相关参数在 json 配置文件中的 `stream` 中配置，具体参数如下。
 
-- **cache** : 缓存块的大小，单位是 MB，默认值是 16。
+- **stream_name** : 流式计算的名称，必填项。
 
-- **blocks** : 每个 vnode 中缓存块的数量，默认为 6。
+- **stream_stb** : 流式计算对应的超级表名称，必填项。
 
-- **precision** : 数据库时间精度，默认值为 "ms"。
+- **stream_sql** : 流式计算的sql语句，必填项。
 
-- **keep** : 保留数据的天数，默认值为 3650。
+- **trigger_mode** : 流式计算的触发模式，可选项。
```

Comment:
trigger_mode must be "at_once" "window_close" "max_delay **s". max_delay unit(**s) could be d/m or others supported by TDenging

Reply:
I will make a link to the stream official page